src/Playwright.Tests.TestServer/SimpleServer.cs

```diff
@@ -163,33 +144,7 @@ public SimpleServer(int port, string contentRoot, bool isHttps)
                             context.Response.StatusCode = StatusCodes.Status304NotModified;
                         }
 
-                        await next().ConfigureAwait(false);
-                    })
-                    .UseMiddleware<SimpleCompressionMiddleware>(this)
-                    .UseStaticFiles(new StaticFileOptions
-                    {
-                        OnPrepareResponse = fileResponseContext =>
-                        {
-                            if (_csp.TryGetValue(fileResponseContext.Context.Request.Path, out string csp))
-                            {
-                                fileResponseContext.Context.Response.Headers["Content-Security-Policy"] = csp;
-                            }
-
-                            if (fileResponseContext.Context.Request.Path.ToString().EndsWith(".html"))
-                            {
-                                fileResponseContext.Context.Response.Headers["Content-Type"] = "text/html; charset=utf-8";
-
-                                if (fileResponseContext.Context.Request.Path.ToString().Contains("/cached/"))
```

Comment:
we don't have a test which access /cached/ files, so removing for now, we can add it back when we need it.